### PR TITLE
Proper rocksdb shutdown if many run_actix threads are running

### PR DESCRIPTION
**Pull request by mm-near**

### CODE REVIEW

**Comment by mm-near** [comment]
passing nayduck: https://nayduck.near.org/#/run/2422

**Review by nikurt — APPROVED** [review]

File: test-utils/actix-test-utils/src/lib.rs [hunk 1]
```diff
@@ -64,8 +68,15 @@ pub fn spawn_interruptible<F: std::future::Future + 'static>(
     actix_rt::spawn(handle_interrupt!(f))
 }
 
+// Number of actix instances that are currently running.
+pub(crate) static ACTIX_INSTANCES_COUNTER: Lazy<Mutex<usize>> = Lazy::new(|| (Mutex::new(0)));
```
Comment (mina86) [inline test-utils/actix-test-utils/src/lib.rs:72]: Just use `AtomicUsize`.

Comment (mm-near) [reply test-utils/actix-test-utils/src/lib.rs:72]: I tried - but then this must be a 'mutable' static variable (as I need access to 'get_mut' - to make sure that I hold the lock while I'm cleaning up the rocksdb handles)

Comment (bowenwang1996) [reply test-utils/actix-test-utils/src/lib.rs:72]: Atomic variables have interior mutability. You can use `load` to read and `store` to update it without requiring the variable to be mutable

**Pull request merged**